pkgs/development/python-modules/crytic-compile/default.nix

```
@@ -19,7 +19,7 @@ buildPythonPackage rec {
     owner = "crytic";
     repo = "crytic-compile";
     rev = "refs/tags/${version}";
-    hash = "sha256-Nx3eKy/0BLg82o3qDHjxcHXtpX3KDdnBKYwCuTLWRUE=";
+    hash = "sha256-b1mAiua+9twXRN6sETednmH24TzZAKOiIIxDSA4mrlc=";
```

Shouldn't it be this ?
```suggestion
    hash = "sha256-CeoACtgvMweDbIvYguK2Ca+iTBFONWcE2b0qUkBbQSU=";
```

---

Thank you! Indeed, I think the upstream changed their artifact. I was probably too fast.